### ISSUE TITLE
how to deal with pip?

### ISSUE THREAD

**Issue by acubed3** [jupyter-server/jupyter-resource-usage#78]
As I understand, it is not possible to install this package from pip and this issue is quite new. How should I deal with this problem? Is it possible to install by hand, not from pip?

**Comment by Zsailer** [comment]
**Installing from PyPI**
This latest version of this package (0.4.0) is still available under its old name, nbresuse. You can install it with `pip`:
```
pip install nbresuse
```

```
jupyter serverextension enable --py nbresuse --sys-prefix
jupyter nbextension install --py nbresuse --sys-prefix
jupyter nbextension enable --py nbresuse --sys-prefix
```

This package will soon be released under a new name, and newer versions (>0.4.0) will be available using the `jupyter-resource-usage` package name.

**Installing from source**

You can install from source by cloning this repo and calling:
```
pip install .
```
at the root of the repository. This will install the package under the *new* jupyter-resource-usage* name.

Then, enable the extension using this series of commands:
```
jupyter serverextension enable --py jupyter-resource-usage --sys-prefix
jupyter nbextension install --py jupyter-resource-usage --sys-prefix
jupyter nbextension enable --py jupyter-resource-usage --sys-prefix
```

**Comment by jtpio** [comment]
@acubed3 FYI `jupyter-resource-usage` is now available on PyPI and conda-forge.

With `pip`:

```bash
pip install jupyter-resource-usage
```

With `conda`:

```bash
conda install -c conda-forge jupyter-resource-usage
```
